### ISSUE TITLE
CGPR1 not working properly

### ISSUE THREAD

**Issue by PhilipsGorniczy** [custom-components/ble_monitor#306]
Hi

I just added CGPR1 `Qingping CGPR1 Qingping Motion & Ambient Light Sensor CGPR1` to hass using ble_monitor. 

Looks like motion is not working. 
![image](https://user-images.githubusercontent.com/80550026/111847082-6a3c1c00-8908-11eb-8c9b-1b02550fdce8.png)

I ran hass in debug:
```
logger:
   default: info
   logs:
     custom_components.ble_monitor: debug
```

And all I was able to get was:
```
home-assistant   | 2021-03-19 23:02:02 DEBUG (MainThread) [custom_components.ble_monitor.binary_sensor] Binary sensor device with mac address 58:2D:34:60:34:B2 has the following settings. Name: QingPingDoorSensor. Restore state: False. Reset Timer: 35
home-assistant   | 2021-03-19 23:02:02 DEBUG (MainThread) [custom_components.ble_monitor.sensor] Sensor device with mac address 58:2D:34:60:34:B2 has the following settings. Name: QingPingDoorSensor. Temperature unit: °C. Decimals: 1. Use Median: False. Restore state: False. Reset Timer: 35.
home-assistant   | 2021-03-19 23:02:02 DEBUG (MainThread) [custom_components.ble_monitor.binary_sensor] async_added_to_hass called for ble motion QingPingDoorSensor
home-assistant   | 2021-03-19 23:02:02 DEBUG (MainThread) [custom_components.ble_monitor.sensor] async_added_to_hass called for ble illuminance QingPingDoorSensor
```

And then the error:
```home-assistant   | 2021-03-19 23:03:01 DEBUG (Thread-3) [custom_components.ble_monitor] HCIdump thread: connected to hci0
home-assistant   | 2021-03-19 23:03:01 DEBUG (Thread-3) [custom_components.ble_monitor] HCIdump thread: start main event_loop
home-assistant   | 2021-03-19 23:04:00 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
home-assistant   | Traceback (most recent call last):
home-assistant   |   File "/config/custom_components/ble_monitor/binary_sensor.py", line 164, in async_run
home-assistant   |     light = sensors[l_i]
home-assistant   | IndexError: list index out of range
home-assistant   | 2021-03-19 23:04:02 DEBUG (MainThread) [custom_components.ble_monitor.sensor] 8 MiBeacon BLE ADV messages processed for 9 measuring device(s).
```

Was anyone able to get motion from this sensor?

**Comment by Ernst79** [comment]
I’ll create a fix for your issue ASAP. It’s an issue in the code of BLE monitor. Some sensors use the message for light on/off, CGPR1 uses the same message type for lux measurements. I think your motion sensor will start working after the fix as well.

**Comment by PhilipsGorniczy** [comment]
Thanks!

**Comment by Ernst79** [comment]
Should be fixed in 1.3.1-beta. Please update to the latest beta and let me know if it is fixed.

**Comment by PhilipsGorniczy** [comment]
I confirm it is working correctly.
Thanks for very fast response!
